From 0c7fe6572ca2835048563d9bed77150d947810b7 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Wed, 14 Mar 2012 19:36:53 +0200
Subject: [PATCH 062/370] fix spelling error: s/casted/cast/

From Webster's Revised Unabridged Dictionary (1913):

Cast \Cast\ (k[.a]st), v. t. [imp. & p. p. Cast; p. pr. & vb.
   n. Casting.] [Cf. Dan. kaste, Icel. & Sw. kasta; perh. akin
   to L. gerere to bear, carry. E. jest.]

Reported-by: Nikita Danilov <nikita_danilov@xyratex.com>
Reviewed-on: http://reviewboard.clusterstor.com/r/714/
---
 conf/confc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/conf/confc.h b/conf/confc.h
index 3b073e9..363d9b1 100644
--- a/conf/confc.h
+++ b/conf/confc.h
@@ -340,7 +340,7 @@ struct c2_confc_ctx {
 	/**
 	 * Pointer to the requested configuration object/directory.
 	 *
-	 * To be type-casted to
+	 * To be type-cast to
 	 * - <tt>struct X *</tt>, where X is a _concrete_ configuration
 	 *   object type (c2_conf_profile, c2_conf_filesystem, etc.),
 	 *   if the field is filled by c2_confc_open() call;
-- 
1.8.3.2

